package.json

```diff
@@ -1,26 +1,23 @@
 {
-    "name": "open-discussions-root",
+    "name": "root",
     "private": true,
     "workspaces": [
         "frontends/*"
     ],
     "scripts": {
-        "foreach-with-fallback": "yarn workspaces foreach --verbose --topological exec node ${PROJECT_CWD}/scripts/yarn-run-with-fallback.js $0 $1",
-        "start-dev": "yarn workspaces foreach --verbose --parallel --interlaced run start-dev",
+        "foreach": "yarn workspaces foreach -pvi --exclude=root run",
```

Comment:
Previously, we were running yarn commands via `yarn-run-with-fallback.js <cmd> <fallback-cmd>`, the idea being that workspaces could define their own `cmd`, and default to a globally-defined `fallback-cmd`.

In https://github.com/mitodl/open-discussions this was useful because we were supporting differing linting/typechecking for old Flow+JS code than for new TS code.

It was a little complicated and is unnecessary now that everything is TS: all projects use the same linting, typechecking, etc commands.

This is much faster: *some* of the speedup comes from not using `yarn-run-with-fallback.js`, most of the speedup comes from running the `foreach` in parallel (`-p`), which we could have been doing any time since
- https://github.com/mitodl/open-discussions/pull/3874
was merged